pages/linux/latexmk.md

```diff
@@ -0,0 +1,19 @@
+# latexmk
+
+> Automated LaTeX document generation.
+
+- Compile .tex file to pdf:
+
+`latexmk -pdf file`
+
+- Force latexmk to generate document despite errors:
+
+`latexmk -f file`
+
+- Clean up files generated by latexmk except dvi, postscript and pdf:
```

Comment:
I am guessing it cleans up files in the current folder ? Would be good to mention it for clarity's sake.